aten/src/ATen/native/cuda/MemoryAccess.cuh

```diff
@@ -67,12 +67,12 @@ struct vectorized_load_helper {
 
 template<int arg_index>
 struct unroll_load_helper {
-  template <typename args_t, typename policy_t, typename offset_t>
-  static __device__ void apply(policy_t &self, args_t *args, offset_t offset, int j, int num_outputs) {
+  template <typename args_t, typename policy_t, typename offset_t, typename loader_t>
+  static __device__ void apply(policy_t &self, args_t *args, offset_t offset, loader_t loader, int j, int num_outputs) {
     using arg_t = std::tuple_element_t<arg_index, args_t>;
     // `data` hold the data_ptr for tensors [output, input0, input1, ...], so we
     // need a +1 offset to get the input
-    auto ptr = reinterpret_cast<arg_t *>(self.data[arg_index + num_outputs]) + offset[arg_index];
+    auto ptr = reinterpret_cast<arg_t *>(loader.template load<arg_t>(self.data[arg_index + 1], offset[arg_index], arg_index));
```

Comment:
wait, should it be `arg_index+num_outputs` instead of `+1` here?

Reply:
Yes, you are right. And fixed.